tests/integ/modin/binary/test_timedelta.py

```diff
@@ -1405,3 +1960,33 @@ def test_two_timedelta_datfaframes(
             (pandas_lhs, pandas_rhs),
             lambda t: getattr(t[0], op_between_timedeltas)(t[1]),
         )
+
+    @sql_count_checker(query_count=1, join_count=1)
+    def test_timedelta_and_numeric(
+        self,
+        timedelta_dataframes_postive_no_nulls_1_2x2,
+        numeric_dataframes_postive_no_nulls_1_2x2,
+        op_between_timedelta_and_numeric,
+    ):
+        snow_lhs, pandas_lhs = timedelta_dataframes_postive_no_nulls_1_2x2
+        snow_rhs, pandas_rhs = numeric_dataframes_postive_no_nulls_1_2x2
+        eval_snowpark_pandas_result(
+            (snow_lhs, snow_rhs),
+            (pandas_lhs, pandas_rhs),
+            lambda t: getattr(t[0], op_between_timedelta_and_numeric)(t[1]),
+        )
+
+    @sql_count_checker(query_count=1, join_count=1)
+    def test_numeric_and_timedelta(
+        self,
+        numeric_dataframes_postive_no_nulls_1_2x2,
+        timedelta_dataframes_postive_no_nulls_1_2x2,
+        op_between_numeric_and_timedelta,
+    ):
+        snow_lhs, pandas_lhs = numeric_dataframes_postive_no_nulls_1_2x2
+        snow_rhs, pandas_rhs = timedelta_dataframes_postive_no_nulls_1_2x2
+        eval_snowpark_pandas_result(
+            (snow_lhs, snow_rhs),
+            (pandas_lhs, pandas_rhs),
+            lambda t: getattr(t[0], op_between_numeric_and_timedelta)(t[1]),
```

Comment:
This is pretty neat, didn't know `eval_snowpark_pandas_result` could handle this